README.md

```diff
@@ -125,25 +133,50 @@ Sometimes you need a service to be available alongside your Jupyter server. In t
 
 A service is described by 
 
-* **template** - a name of a template imported in OpenShift which describes how to deploy the service
-* **parameters** - an object containing key-value pairs to be processed into the template
+* **resources** - these are the names of configMaps which contain various templates. For each resource there can be a different configMap with a different path. Or we can use a single configMap containing all templates and these are simply taken by using the correct path.
+* **configuration** - an object containing key-value pairs to be processed into the template
```

Comment:
This is just a note and we will not implement it here, but just a food for thought - we might want to try to figure out how to reference CMs and Secrets here since that would allow users to share secrets into services